zcash_primitives/src/zip32.rs

```diff
@@ -409,6 +446,38 @@ impl ExtendedSpendingKey {
     pub fn default_address(&self) -> (DiversifierIndex, PaymentAddress) {
         ExtendedFullViewingKey::from(self).default_address()
     }
+
+    pub fn derive_internal(&self) -> Self {
```

Comment:
Document this (yes the existing documentation is sparse; we should improve it as we go).